### PR TITLE
remove util package as webpack 5 does not like

### PR DESCRIPTION
**Pull request by muselesscreator**
**TL;DR -**
Removes is an out-dated requirement of the StrictDict tool, and is incompatible with Webpack 5.
Alerted to this by David J, who has already had to make this change in the learning MFE

**Developer Checklist**
- [ ] Test suites passing
- [ ] Documentation and test plan updated, if applicable
- [ ] Received code-owner approving review
- [ ] Bumped version number [package.json](../package.json)

**Testing Instructions**

[ How should a reviewer test this PR? ]

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

**Review by nsprenkle — APPROVED** [review]
Read through and highest levels "is it melting down or nah" tests look good. :shipit:

**Review by leangseu-edx — APPROVED** [review]

**Comment by jansenk** [comment]
This is a real old PR. At this point the only change is that we're removing 'util' as a requirement. Since this is a PR form a fork I can't commit to the original branch, so I id the merge and stuff into this other PR: https://github.com/openedx/frontend-app-gradebook/pull/304
